### ISSUE TITLE
apt-get update fails, and therefore apt-cache search

### ISSUE THREAD

**Issue by ouboub** [fink/fink#223]
Hi 
when running 
sudo /opt/sw/bin/apt-get update. (BTW where can I set the patch in MacOS?)
I obtain

Err http://bindist.finkmirrors.net stable/main Packages                     
  404 Not Found
Get:1 http://bindist.finkmirrors.net stable/main Release [85B]
Fetched 85B in 0s (177B/s) 
Failed to fetch http://bindist.finkmirrors.net/10.15/dists/stable/main/binary-darwin-x86_64/Packages  404 Not Found
Reading Package Lists... Done
Building Dependency Tree... Done
E: Some index files failed to download, they have been ignored, or old ones used instead.

I also tried
sudo ./apt-get install git
Reading Package Lists... Done
Building Dependency Tree... Done
W: Couldn't stat source package list http://bindist.finkmirrors.net stable/main Packages (/opt/sw/var/lib/apt/lists/bindist.finkmirrors.net_10.15_dists_stable_main_binary-darwin-x86%5f64_Packages) - stat (2 No such file or directory)
W: Couldn't stat source package list http://bindist.finkmirrors.net stable/main Packages (/opt/sw/var/lib/apt/lists/bindist.finkmirrors.net_10.15_dists_stable_main_binary-darwin-x86%5f64_Packages) - stat (2 No such file or directory)
W: You may want to run apt-get update to correct these problems
E: Couldn't find package git

**Comment by nieder** [comment]
There's no binary distribution for 10.15. As mentioned in fink/scripts#55, edit the file `/opt/sw/etc/fink.conf` and set the field `UseBinaryDist` to `false`.

**Comment by ouboub** [comment]
ok, thanks, that would allow me then to use apt-get (as I am used to)?

**Comment by nieder** [comment]
Not really. apt-get only works with .deb archives. Since 10.15 has no binary distribution, you'll need to use the `fink` tool to build packages from source.

**Comment by dmacks** [comment]
Is there someone working on restarting the builder box to create a 10.15 .deb archive? Should we have this be a non-fatal warning (especially at fink's own setup/installation-time)?

**Comment by nieder** [comment]
I think @bcully ran the buildbots, but he stopped some time ago.
You're right it should probably be a warning anytime, rather than an error.

**Comment by TheSin-** [comment]
I’ve been wanted to, I have the scripts from Brandon just haven’t had time to setup the VMs.  I made changes yesterday to hopefully make it non fatal I didn’t have time to test so if it still is I’ll fix it tonight to be none fatal.
